features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/ImageGenerator/DocThumb.jsx

```diff
@@ -120,7 +120,7 @@ class DocThumb extends Component {
                         <Grid item md={6}>
                             <FormattedMessage
                                 id='Apis.Listing.components.ImageGenerator.DocThumb.apiVersion'
-                                defaultMessage='Api Version'
+                                defaultMessage='API Version'
```

Comment:
API version?

Reply:
Leaving this as it is, because similar messages follow same format.